book/src/runtime.md

```diff
@@ -27,27 +29,33 @@ entrypoint.
 
 ## Execution
 
-Transactions are batched and processed in a pipeline
+Transactions are batched and processed in a pipeline.  The TPU and TVU follow a
+slightly different path.  The TPU runtime ensures that PoH record occurs before
+memory is committed.
 
-<img alt="Runtime pipeline" src="img/runtime.svg" class="center"/>
+<img alt="Runtime pipeline" src="img/tpu-runtime.svg" class="center"/>
+
+The TVU runtime ensures that PoH verification occurs before the runtime
+processes any transactions.
+
+<img alt="Runtime pipeline" src="img/tvu-runtime.svg" class="center"/>
 
 At the *execute* stage, the loaded pages have no data dependencies, so all the
```

Comment:
"Pages" should be defined somewhere or referred to in a consistent manner if multiple defined.

Reply:
@jackcmay ill purge pages :)